modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,14 +5,109 @@ This release specifically highlights agility, performance, and manageability and
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements that are available in this release.
 
+[#release-555]
+== Release 5.5.5
+
+Couchbase Server 5.5.5, released in July 2019, is the fifth maintenance release in the 5.5.x series for Couchbase Server.
+This maintenance release includes fixes to a number of components.
+
+[#fixed-issues-555]
+=== Fixed Issues
+
+==== Cross Datacenter Replication (XDCR)
+
+[#table_fixedissues_xdcr_555,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-34297[MB-34297^]
+| The topology change detector, upon detecting that a target bucket no longer exists, would delete the replication spec right away. However, in case the target node was being failed over adn had stopped serving the bucket, this could cause the replication spec to be incorrectly deleted. This has been fixed and the topology change detector does not delete replication spec when the target bucket cannot be found.
```

Comment:
spell mistake - adn instead of and

Reply:
> the topology change detector does not delete replication spec when the target bucket cannot be found

XDCR no longer deletes replication specs incorrectly during transient rebalancing states.

_Explanation: If the target bucket is removed, XDCR still will remove the replication spec using a non-topology-change-detector module. This removal mechanism is much more fail-safe._